src/routes/DigitalAssets/routes/Send/sagas/digitalAssetsSend.js

```diff
@@ -499,6 +525,10 @@ function* setPriority(): Saga<void> {
     priority,
   }: ExtractReturn<typeof selectDigitalAssetsSend> = yield select(selectDigitalAssetsSend)
 
+  // force reset warnings, when asset address changed
```

Comment:
> when asset address changed

`function* setPriority`

Reply:
will be fixed in next pr (task: estimate gas fetch)